proof/proofplayer/src/TOutputListSelectorDataMap.cxx

```diff
@@ -30,7 +30,7 @@ output list.
 
 #include <cstddef>
 
-namespace {
+namespace AN {//anonymous namespace
```

Comment:
I think @gganis should tell if that's correct or not.

Reply:
I guess this might be another case of doxygen giving spurious errors, I can revert it if you think it's better as before.

Reply:
That's code ... This code is working, there is not bug report bon it ... I would not change it ... but @gganis is the author. It is up to him to approuve. 
Note that it would be could good to not mix changes in "comments" which every can approuve and code changes which need author approval.

Reply:
Yes, you are right. I reverted it :)